src/components/modal/Modal.scss

```diff
@@ -5,6 +5,8 @@
 /**************************************
  * Modal
  **************************************/
+$responsive-modal-header-height: 64px;
+$responsive-modal-actions-height: 60px;
```

Comment:
Can we move this inside the `@include` block since it's specific to those styles?
https://github.com/box/box-ui-elements/blob/96ee050f3928afef1644d48b2bea296a052914cd/src/components/modal/Modal.scss#L143

Reply:
Done